mindmeld/components/dialogue.py

```diff
@@ -90,8 +90,8 @@ def __init__(self, dialogue_state, **kwargs):
         Args:
             dialogue_state (str): The name of the dialogue state.
             domain (str): The name of the domain to match against.
-            has_entity (str, list, set): A synonym for the ``has_entities`` param.
-            has_entities (str, list, set): A single entity type or a list of entity types to match
+            has_entity (str): A single entity type to match.
+            has_entities (str, list, set): A single entity type or a list/set of entity types to match
```

Comment:
has_entities shouldn't be a string, because it will turn the string into a set of characters

Reply:
Done